docs/getting-started-guides/aws.md

```diff
@@ -93,10 +93,10 @@ sudo cp kubernetes/platforms/darwin/amd64/kubectl /usr/local/bin/kubectl
 sudo cp kubernetes/platforms/linux/amd64/kubectl /usr/local/bin/kubectl
 ```
 
-An up-to-date documentation page for this tool is available here: [kubectl manual](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/kubectl.md)
+An up-to-date documentation page for this tool is available here: [kubectl manual](../../docs/user-guide/kubectl/kubectl.md)
```

Comment:
We should at least understand this lest it come back to bite us in the tush.

On Thu, Jul 16, 2015 at 4:29 PM, Daniel Smith <notifications@github.com>
wrote:

> In docs/getting-started-guides/aws.md
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/11400#discussion_r34849645>
> :
>
> > @@ -93,10 +93,10 @@ sudo cp kubernetes/platforms/darwin/amd64/kubectl /usr/local/bin/kubectl
> >  sudo cp kubernetes/platforms/linux/amd64/kubectl /usr/local/bin/kubectl
> >  ```
> >
> > -An up-to-date documentation page for this tool is available here: [kubectl manual](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/kubectl.md)
> > +An up-to-date documentation page for this tool is available here: [kubectl manual](../../docs/user-guide/kubectl/kubectl.md)
>
> Empirically, it seems stable... but that's not what I expect it to do. Not
> going to solve this mystery right now.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/11400/files#r34849645>
> .
>